doc/monitoring/getting_started.rst

```diff
@@ -194,3 +194,120 @@ via configuration.
+    To start collecting metrics, add a `job <https://prometheus.io/docs/prometheus/latest/getting_started/#configure-prometheus-to-monitor-the-sample-targets>`_
+    to Prometheus configuration with each Tarantool instance URI as a target and
+    metrics path as it was configured on Tarantool instances:
+
+    ..  code-block:: yaml
+
+        scrape_configs:
+          - job_name: "example_project"
+            static_configs:
+              - targets: 
+                - "example_project:8081"
+                - "example_project:8082"
+                - "example_project:8083"
+            metrics_path: "/metrics/prometheus"
+
+    To collect metrics for InfluxDB, you must use Telegraf agent.
+    First off, configure Tarantool metrics output in ``json`` format
+    with :ref:`cartridge.roles.metrics <cartridge-role>` configuration or
+    corresponding :ref:`output plugin <json>`. To start collecting metrics,
+    add `http input <https://github.com/influxdata/telegraf/blob/release-1.17/plugins/inputs/http/README.md>`_
+    to Telegraf configuration including each Tarantool instance metrics URL:
+
+    ..  code-block:: text
+
+        [[inputs.http]]
+            urls = [
+                "http://example_project:8081/metrics/json",
+                "http://example_project:8082/metrics/json",
+                "http://example_project:8083/metrics/json"
+            ]
+            timeout = "30s"
+            tag_keys = [
+                "metric_name",
+                "label_pairs_alias",
+                "label_pairs_quantile",
+                "label_pairs_path",
+                "label_pairs_method",
+                "label_pairs_status",
+                "label_pairs_operation"
+            ]
+            insecure_skip_verify = true
+            interval = "10s"
+            data_format = "json"
+            name_prefix = "example_project_"
+            fieldpass = ["value"]
+
+    Be sure to include each label key as ``label_pairs_<key>`` so it will be
+    extracted with plugin. For example, if you use :code:`{ state = 'ready' }` labels
+    somewhere in metric collectors, add ``label_pairs_state`` tag key.
+
+    If you connect Telegraf instance to InfluxDB storage, metrics will be stored
+    with ``"<name_prefix>http"`` measurement (``"example_project_http"`` in our example).
+
+#.  Import a dashboard from `Grafana Official & community built dashboards <https://grafana.com/grafana/dashboards>`_.
+
+    Open Grafana import menu.
+
+    ..  image:: images/grafana-import-v6.png
+        :align: center
+
+    To import specific dashboard, choose one of the following options:
+
+    - paste dashboard id (``12567`` for InfluxDB dashboard, ``13054`` for Prometheus dashboard), or
+    - paste link to dashboard (https://grafana.com/grafana/dashboards/12567 for InfluxDB dashboard,
+      https://grafana.com/grafana/dashboards/13054 for Prometheus dashboard), or
+    - paste dashboard json file contents, or
```

Comment:
is it possible to provide direct link to download json file?

Reply:
It seems to be possible, according to the Sphinx docs ([:download: role](https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#referencing-downloadable-files)), but we never tried

Reply:
It is possible (e.g. here https://github.com/tarantool/grafana-dashboard/releases/tag/0.2.2) to make a link to specific revision, so we will need to update it every time I upload a new version.